torchtext/data/field.py

```diff
@@ -196,3 +199,54 @@ def numericalize(self, arr, device=None, train=True):
         if self.include_lengths:
             return Variable(arr, volatile=not train), lengths
         return Variable(arr, volatile=not train)
+
+
+class ReversibleField(Field):
+
+    def __init__(self, **kwargs):
+        if kwargs.get('tokenize') not in ('revtok', 'subword'):
+            kwargs['tokenize'] = 'revtok'
+        super(ReversibleField, self).__init__(**kwargs)
+
+    def reverse(self, batch):
+        try:
+            import revtok
+        except ImportError:
+            print("Please install revtok.")
+            raise
+        if not self.batch_first:
+            batch.t_()
```

Comment:
Using t() would allow this to work on variables and tensors, which seems to be desirable based on the TREC example you wrote for subwords

Reply:
Decision was to not support variables in reverse: ac4f794 alters the test that broke on this line